SubmittingPatches.rst

```diff
@@ -519,3 +513,134 @@ decreasing the likelihood of your MIME-attached change being accepted.
 Exception: If your mailer is mangling patches then someone may ask
 you to re-send them using MIME.
 
+
+BACKPORTING
+===========
+
+Ceph has a `Stable Releases and Backports`_ team, staffed by volunteers,
+which is charged with maintaining the stable releases and backporting bugfixes
+from the master branch to them.
+
+Even if your objective is to get a bugfix into a stable release, your initial
+PR still needs to target master.
+
+.. _`Stable Releases and Backports`: http://tracker.ceph.com/projects/ceph-releases/wiki
+
+Backport tracking
+-----------------
+
+Bugfixes for which backports are needed/desired need to be tracked.
+Anyone can open an account at https://tracker.ceph.com and open bugs in the
+"Ceph" project. When opening the bug, fill in the "Backport" field with a
+comma- or space-separated list of releases, e.g. ::
+
+    Backport: mimic,luminous
+
+This will notify other developers that your bugfix should be cherry-picked to
+one or more stable branches after it has been merged into master.
+
+(If the Backport field is empty and you don't have
+sufficient Redmine privileges to fill it, just leave a comment asking for what
+you want done, and one of the developers will follow up.)
+
+Then, target the "master" branch in your pull request. Make sure you reference
+the tracker issue in the commit message of your fix. This is covered in the
+section `Tag the commit`_.
+
+When the pull request is merged, the developer who performed the merge will
+change the tracker status to "Pending Backport" and, in due course, backport
+tracker issues will be created for each of the stable branches for which
+backports were requested in the issue. If you get tired of waiting for the
+backport tracker issues to appear, ping the reviewers in the PR or on IRC.
+
+Backport staging
+----------------
+
+Backports can be staged for bugfixes that have been merged into master,
+provided the relevant backport tracker issues have been created. Do not stage
+backport PRs for any commit that has not yet been merged to master.
+
+Role of backporting team
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Ordinarily, it is enough to fill out the "Backport" field in the bug (tracker
+issue). The volunteers from the Stable Releases and Backports team will
+backport the fix, run regression tests on it, and include it in one or more
+future point releases.
+
+Sometimes bugfix authors are motivated to stage the backport PR(s) themselves.
+Here's how to do that.
+
+Cherry-picking instructions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you want to stage the backport yourself, first find out the backport tracker
+number of the fix (see `Backport tracking`_, above) by referring to the
```

Comment:
I don't see the part that describes how the Backport tracker item is created.  (I'm unsure how these are created mysefl!  Is there a script that does this?)

Reply:
Yes, the script is here: https://github.com/ceph/ceph/blob/master/src/script/backport-create-issue

I did not mention it on purpose because I didn't want to encumber `SubmittingPatches.rst` with Too Much Information, especially seeing as new contributors typically do not have sufficient tracker permissions to run the script.

But I agree this information should be presented *somewhere*. Here?